### PR TITLE
Fix docker command for clean task

### PR DESCRIPTION
**Pull request by Nuvindu**
## Purpose
Fixes: https://github.com/ballerina-platform/plugin-gradle/issues/131

### CODE REVIEW

File: src/main/groovy/io/ballerina/plugin/BallerinaPlugin.groovy [hunk 1]
```diff
@@ -414,9 +414,8 @@ class BallerinaPlugin implements Plugin<Project> {
                         def deleteUsingDocker = """
                             docker run -u root \
                                 -v $parentDirectory:/home/ballerina/$parentDirectory.name \
-                                -v $projectDirectory:/home/ballerina/$parentDirectory.name/$projectDirectory.name \
                                 ballerina/ballerina:$ballerinaDockerTag \
-                                /bin/sh -c "cd $parentDirectory.name/$projectDirectory.name && rm -rf build target"
+                                /bin/sh -c "cd /home/ballerina/$parentDirectory.name && rm -rf build target ballerina/build ballerina/target"
```
Comment (DimuthuMadushan) [inline src/main/groovy/io/ballerina/plugin/BallerinaPlugin.groovy:418]: If there are other directories that include `target` and `build` directories, we have to add those as well.  Maybe we can try out something like following. `find . -type d \( -name "target" -o -name "build" \) -exec rm -r {} +`

Comment (Nuvindu) [reply src/main/groovy/io/ballerina/plugin/BallerinaPlugin.groovy]: This command returns a syntax error at `\(`. Therefore I used two separate commands to delete 'build' and 'target' directories

Comment (DimuthuMadushan) [reply src/main/groovy/io/ballerina/plugin/BallerinaPlugin.groovy]: Can't we use `&&` operation to combine the two commands?

**Review by DimuthuMadushan — APPROVED** [review]
LGTM

**Pull request merged**